modules/acquia_cms_common/tests/src/Kernel/AcquiaCmsTelemetryTest.php

```diff
@@ -129,16 +157,18 @@ public function testAcquiaCmsTelemetryDataWithSiteStudio(): void {
       ->set("use_dx8", "enable")
       ->set("api_key", "some-random-key")
       ->save(TRUE);
-
     $expected_telemetry_data = [
       "acquia_cms" => [
         "application_uuid" => "",
         "application_name" => "",
         "environment_name" => "",
+        "acsf_status" => FALSE,
+        "site_uri" => $this->siteUri,
+        "site_name" => $this->siteName,
         "starter_kit_name" => "acquia_cms_existing_site",
         "starter_kit_ui" => FALSE,
         "site_studio_status" => TRUE,
-        "profile" => '',
+        "profile" => "",
```

Comment:
@rajeshreeputra Why profile is empty in test case ?

Reply:
Kernel event with testing profile is sending profile name as blank, further investigation in progress.